@here/harp-omv-datasource/lib/OmvDebugLabelsTile.ts

```diff
@@ -150,62 +202,66 @@ export class OmvDebugLabelsTile extends OmvTile {
-                                const i = textPath.path.length - 2;
-                                const x = textPath.path[i];
-                                const y = textPath.path[i + 1];
-                                const z = 0;
-                                // give path point a simple geometry: a diamond
-                                const circleGeometry = new THREE.CircleGeometry(2, 4);
-                                circleGeometry.scale(3, 3, 3);
-                                circleGeometry.translate(x, y, z);
-                                const cmesh = new THREE.Mesh(
-                                    circleGeometry,
-                                    debugCircleMaterial2WF
-                                );
-                                cmesh.renderOrder = 3000;
-                                this.objects.push(cmesh);
-                            }
+                    // the lines of a path share a common geometry
+                    const N = textPath.path.length / 3;
+                    for (let i = 0; i < N; ++i) {
+                        if (i > 0) {
+                            lineIndices.push(baseVertex + i);
                         }
-
-                        // the lines of a path share a common geometry
-                        const N = textPath.path.length / 2;
-                        for (let i = 0; i < N; ++i) {
-                            if (i > 0) {
-                                indices.push(baseVertex + i);
-                            }
-                            if (i + 1 < N) {
-                                indices.push(baseVertex + i);
-                            }
+                        if (i + 1 < N) {
+                            lineIndices.push(baseVertex + i);
                         }
-
-                        geometry.addGroup(0, indices.length, 0);
-                        geometry.addAttribute(
-                            "position",
-                            new THREE.BufferAttribute(new Float32Array(positions), 3)
-                        );
-                        geometry.setIndex(new THREE.BufferAttribute(new Uint32Array(indices), 1));
-                        const mesh = new THREE.LineSegments(geometry, debugMaterial);
-                        mesh.renderOrder = 2000;
-                        this.objects.push(mesh);
                     }
                 }
             }
+
+            if (lineIndices.length > 0) {
+                lineGeometry.addGroup(0, lineIndices.length, 0);
+
+                lineGeometry.addAttribute(
+                    "position",
+                    new THREE.BufferAttribute(new Float32Array(linePositions), 3)
+                );
+
+                lineGeometry.setIndex(new THREE.BufferAttribute(new Uint32Array(lineIndices), 1));
+                const lineMesh = new THREE.LineSegments(lineGeometry, debugMaterial);
+                lineMesh.renderOrder = 2000;
```

Comment:
This is not new here, but where are these `renderOrder` values coming from? Don't 2000 / 2500 / 3000 look a bit random? What about having a `baseDebugRenderOrder` and adding 1 / 2 / 3 ?